heroku.yml

```diff
@@ -0,0 +1,9 @@
+build:
+  docker:
+    index-regen: heroku.Dockerfile
+run:
+  index-regen: python main.py
+release:
+  image: index-regen
```

Comment:
Maybe `indexer` rather than `index-regen` to be consistent (and because the chain is determined by env vars).

Reply:
👌

Reply:
5fab0dbd67735c6a8fb7be2759278507d0522929

Reply:
5fab0dbd67735c6a8fb7be2759278507d0522929